lib/Runtime/Language/ModuleNamespace.cpp

```diff
@@ -338,28 +330,4 @@ namespace Js
         }
         return sortedExportedNames;
     }
-
-    Var ModuleNamespace::EntrySymbolIterator(RecyclableObject* function, CallInfo callInfo, ...)
```

Comment:
So without this, does property iteration still work just by doing an ordinary walk across the type handler?